### ISSUE TITLE
Add Ignores per Project

### ISSUE THREAD

**Issue by chillvetica** [atom/whitespace#138]
Would be nice if we could selectively disable the whitespace package for certain projects. Occasionally need to work in a repo with very old code and :tons: of whitespace issues, but don't want to have to keep turning this on and off for old repos vs new repos.

Does this functionality already exist?

**Comment by emileber** [comment]
A workaround while waiting for this feature is to stage only the lines you explicitly changed, and ignore lines changed by this plugin.

**Comment by chillvetica** [comment]
Agreed, just adds a layer of complexity to workflows. I primarily use `git add -p` and there can be hundreds of whitespace changes to sift through. No big deal to jump into a GUI app, but still, would be nice if it could be toggled.
